pycrostates/utils/_docs.py

```diff
@@ -11,21 +11,29 @@
 docdict = dict()
 
 # ---- Documentation to inc. from MNE ----
-keys = ['random_state', 'verbose',
-        'reject_by_annotation_raw', 'n_jobs',
-        'tmin_raw', 'tmax_raw', 'picks_all']
+keys = (
+    'n_jobs',
+    'picks_all',
+    'random_state',
+    'tmin_raw',
+    'tmax_raw',
+    'reject_by_annotation_raw',
+    'verbose',
+    )
 
 for key in keys:
     docdict[key] = docdict_mne[key]
-# TODO: sphinx :term:`data channels` in 'picks_all' has to be included.
+
+# TODO: sphinx :term:`data channels` in 'picks_all' to be inc.
+# TODO: :ref:`logging documentation <tut-logging>` in 'verbose' to be inc.
 
 # ---- Clusters ----
 docdict['fit_inst'] = """
-inst : `~mne.io.Raw` | `~mne.Epochs`
+inst : Raw | Epochs
     Instance containing data to transform to cluster-distance space
     (absolute spatial correlation)."""
 docdict['predict_inst'] = """
-inst : `~mne.io.Raw` | `~mne.Epochs`
+inst : Raw | Epochs
```

Comment:
Same as for Figure, Raw, Epochs, those names should be resolved by numpydoc directly.